contracts/test/TestTransfer.sol

```diff
@@ -40,21 +40,23 @@ contract TestTransfer is Transfer {
     }
 
     function testProcessTransferCommit(
-        bytes32 stateRoot,
+        bytes32 postStateRoot,
+        bytes32 previousStateRoot,
```

Comment:
```suggestion
        bytes32 postStateRoot,
        bytes32 preStateRoot,
```
`preStateRoot` looks nicer next to `postStateRoot` to me. Also I'd put _pre_ before _post_.